make/git/git_pullall.sh

```diff
@@ -44,6 +44,8 @@ if test -z "${has_remote}"; then
 else
     echo "First, updating with rebase from remote ${branch}."
     git pull --rebase origin ${branch}
+    echo "\nSecond, updating submodules."
```

Comment:
This will print a literal `\n`.  Use -e to interpret newlines.

Reply:
Done.